ztp/source-crs/StoragePVC.yaml

```
@@ -0,0 +1,15 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvc-storage
+  namespace: default
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+    volume.beta.kubernetes.io/storage-class: $storageclass
+# User should supply all required content
+spec: {}
+  #accessModes:
+  #  - ReadWriteOnce
+  #resources:
+  #  requests:
+  #    storage: 400Gi
```

I think it's better to have a fully-formed example, if that doesn't break anything:
```suggestion
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 400Gi
```

---

The issue comes in if they don't want to include those sub-sections. We don't have a way of removing them.